.devcontainer/createCommand.sh

```diff
@@ -4,9 +4,14 @@ set -e
 
 echo "Fixing permissions"
 
-sudo chown demisto .venv
-sudo chown demisto node_modules
 sudo chown demisto /workspaces
+sudo chown demisto /workspaces/content
+sudo chown -R demisto /workspaces/content/.vscode
+sudo chown -R demisto /workspaces/content/.git
+sudo chown -R demisto /workspaces/content/.venv
+sudo chown -R demisto /workspaces/content/node_modules
+sudo chown -R demisto /workspaces/content/package-lock.json
+
```

Comment:
```suggestion
sudo chown -R demisto /workspaces/content
```

Reply:
This heavily reduces the performance, as it take time to change the ownership of all the content repo.
Those are the directories that needed for the environment installation, which the permissions are not set for them yet.

For some reason, after the `onCreateCommand`, the `demisto` user in the container has writable access to all the workspace, just during this command the permissions are missing.

Reply:
```suggestion
sudo chown -R demisto /workspaces/content /workspaces/content/.vscode /workspaces/content/.git demisto /workspaces/content/.venv /workspaces/content/node_modules /workspaces/content/package-lock.json
```
`chown` accepts multiple folder / file parameters